SyntaxBasicsHomework/WriteDuodecimalNumbersInRange/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace WriteDuodecimalNumbersInRange
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Number a = ");
+            int a = int.Parse(Console.ReadLine());
+
+            Console.Write("Number b = ");
+            int b = int.Parse(Console.ReadLine());
+
+            for (int i = a; i <= b; i++)
+            {
+                string duodecimal = "";
+                while (i > 0)
+                {
+                    int remainder = i % 12;
+                    if (remainder == 10)
+                        duodecimal = "A" + duodecimal;
+                    else if (remainder == 11)
+                        duodecimal = "B" + duodecimal;
+                    else
+                        duodecimal = remainder.ToString() + duodecimal;
+
+                    i /= 12;
+                }
+                if (duodecimal.Count(c => c == 'A') == 2)
+                {
```

Comment:
You convert 10 to 12-base system and back.. For what? It is not needed for the task. If you even do this please move this functionality to separate methods